cmd/run.go

```diff
@@ -580,6 +587,31 @@ func runL2ClientIfNeeded(ctx context.Context,
 	return l2Client
 }
 
+func runReorgDetectorL1IfNeeded(
+	ctx context.Context,
+	components []string,
+	l1Client aggkittypes.BaseEthereumClienter,
+	cfg *reorgdetector.Config,
+) (*reorgdetector.ReorgDetector, chan error) {
+	if !isNeeded([]string{
+		aggkitcommon.AGGORACLE, aggkitcommon.AGGSENDER, aggkitcommon.AGGSENDERVALIDATOR,
+		aggkitcommon.BRIDGE, aggkitcommon.AGGCHAINPROOFGEN, aggkitcommon.L1BRIDGESYNC},
```

Comment:
Why do we need the L1 reorg detector for the `AGGORACLE`, `AGGSENDER`, `AGGSENDERVALIDATOR` etc.? Am I missing something out?

```suggestion
		aggkitcommon.L1BRIDGESYNC},
```

Reply:
fixed: 7ad667f85272bca8cbb655f5c8ea9eaff736952a